src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  transition: all 0.2s;
+
+  &:hover {
+    transform: scale(1.2);
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    transition: color 0.2s;
+    color: $main-accent-color;
+  }
+
+  &:hover &__name {
+    color: #34568b;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $secondary-color;
+  }
+
+  &__info {
+    @include flex-center();
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-accent-color;
+  }
+
+  &__price {
+    @include flex-center();
+    margin-bottom: 16px;
+    color: $secondary-color;
+
+    &-title {
+      font-size: 12px;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      color: $main-accent-color;
+    }
+  }
+
+  &__buy {
+    display: inline-block;
+    height: 40px;
+    width: 100%;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    text-align: center;
+    color: #fff;
+    padding: 12px 70px;
```

Comment:
The padding for the buy button is set to 12px 70px, which might cause the text to overflow or the button to be too wide, especially on smaller screens. Consider using relative units or media queries to adjust the padding for different screen sizes.